### PR TITLE
Implementing display solution for testnets in the network dropdown

### PR DESCRIPTION
**Pull request by ryanml**
Designs: https://www.figma.com/file/F8NVM9pM8RpIe3YNbHsk5r/Switch-Network?node-id=514%3A683

Screen Shot 2021-11-02 at 1 14 06 PM
Screen Shot 2021-11-02 at 1 13 15 PM

With dismiss capability: (Added)
Screen Shot 2021-11-05 at 11 22 25 AM
Screen Shot 2021-11-05 at 11 16 58 AM

### CODE REVIEW

File: app/_locales/en/messages.json [hunk 1]
```diff
@@ -594,7 +594,8 @@
     "message": "Decrypt request"
   },
   "defaultNetwork": {
-    "message": "The default network for Ether transactions is Mainnet."
+    "message": "The default network for Ether transactions is Mainnet. You can also $1 testnets $2.",
+    "description": "$1 is the 'enable' or 'disable' key, depending on whether the display of test networks is enabled or not. $2 is a clickable link with text defined by the 'here' key. The link will open to the advanced settings where users can enable the display of test networks in the network dropdown."
```
Comment (danjm) [inline app/_locales/en/messages.json:598]: Hmmm.....`You can also $1 testnets $2.` won't appear in the translations when we ship :thinking: 

So should we include it in english in other languages?

Comment (ryanml) [reply app/_locales/en/messages.json:598]: Sounds good to me, this translation will just fallback to english now

**Review by darkwing — CHANGES_REQUESTED** [review]
This looks excellent!  Just one small nit:  the other headings start with action words:

ActionWords

Maybe this should be "Show test networks"?

**Comment by ryanml** [comment]
@darkwing agreed! I think it should be `Show test networks` as well. cc: @yanrong-chen

**Review by darkwing — DISMISSED** [review]

**Review by NiranjanaBinoy — DISMISSED** [review]

File: ui/components/app/dropdowns/network-dropdown.js [hunk 2]
```diff
@@ -294,27 +309,48 @@ class NetworkDropdown extends Component {
...
+          {showTestnetMessageInDropdown ? (
+            <div className="network-dropdown-content">
+              {t('defaultNetwork', [
+                <span key="testNetworksEnabled">
+                  {shouldShowTestNetworks ? t('disable') : t('enable')}
+                </span>,
+                <span
+                  key="advancedSettingsLink"
+                  className="network-dropdown-content--link"
+                  onClick={() => {
```
Comment (Gudahtt) [inline ui/components/app/dropdowns/network-dropdown.js:323]: Please do not use spans for buttons! This makes them impossible to focus with a keyboard. Semantic HTML is accessible HTML.

Comment (ryanml) [reply ui/components/app/dropdowns/network-dropdown.js:323]: this was an oversight, thanks for pointing it out, fix pushed

Comment (ryanml) [reply ui/components/app/dropdowns/network-dropdown.js:323]: woops, one more thing to push

Comment (darkwing) [reply ui/components/app/dropdowns/network-dropdown.js:323]: Good catch!  Sorry I didn't see this!

Comment (ryanml) [reply ui/components/app/dropdowns/network-dropdown.js:323]: updated thanks!

**Review by NiranjanaBinoy — APPROVED** [review]

**Review by segun — APPROVED** [review]
LGTM

**Comment by darkwing** [comment]
This is good. The only thing I noticed that we should standardize is "testnet" vs. "test network":
testnet

**Comment by ryanml** [comment]
@darkwing - makes sense to me. Should we confer with design and do this in a separate PR? cc: @yanrong-chen

**Comment by ryanml** [comment]
My preference is "test networks"

**Review by darkwing — APPROVED** [review]

**Pull request merged**